### PR TITLE
[arm] Add manifest steps to base-images

### PR DESCRIPTION
**Pull request by jonathanmetzman**
Related: #8164

### CODE REVIEW

**Comment by jonathanmetzman** [comment]
/gcbrun trial_build.py skcms log4j2 --sanitizer address memory undefined coverage --fuzzing-engine libfuzzer afl honggfuzz --force-build

File: infra/build/functions/base_images.py [hunk 1]
```diff
@@ -110,6 +110,48 @@ def run_build(steps, images, tags=None, build_version=MAJOR_TAG):
                              use_build_pool=False)
 
 
+def get_arm_manifest_steps():
```
Comment (oliverchang) [inline infra/build/functions/base_images.py:113]: nit: add docstrings for these functions.

File: infra/build/functions/base_images.py [hunk 2]
```diff
@@ -110,6 +110,48 @@ def run_build(steps, images, tags=None, build_version=MAJOR_TAG):
...
+  images = [f'{TAG_PREFIX}/base-builder', f'{TAG_PREFIX}/base-runner']
+  steps = []
+  for image in images:
+    steps.extend(get_push_manifest_steps(image))
+  return steps
+
+
+def get_push_manifest_steps(image):
+
+  arm_version = f'{image}-testing-arm'
```
Comment (oliverchang) [inline infra/build/functions/base_images.py:123]: What's the difference between arm and arm64 in this context?

Comment (jonathanmetzman) [reply infra/build/functions/base_images.py:123]: `-testing-arm` refers to the image I built on GCE and pushed manually. I renamed them arm_testing_image and arm64_manifest_image.

**Comment by jonathanmetzman** [comment]
/gcbrun trial_build.py skcms log4j2 --sanitizer address memory undefined coverage --fuzzing-engine libfuzzer afl honggfuzz --force-build

**Comment by jonathanmetzman** [comment]
/gcbrun trial_build.py skcms log4j2 --sanitizer address memory undefined coverage --fuzzing-engine libfuzzer afl honggfuzz --force-build

**Comment by jonathanmetzman** [comment]
/gcbrun trial_build.py skcms --sanitizer address memory undefined coverage --fuzzing-engine libfuzzer afl honggfuzz --force-build

**Pull request merged**